app/scripts/require_config.js

```diff
@@ -27,6 +27,7 @@ require.config({
     chai: '../bower_components/chai/chai',
     cocktail: '../bower_components/cocktail/Cocktail',
     crosstab: 'vendor/crosstab',
+    durationJs: '../bower_components/Duration.js/duration',
```

Comment:
we can probably just call it `duration` here and just `require('duration')`, `durationJs` looks a bit ugly.